lib/ansible/modules/windows/win_mssql_login.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Daniele Lazzari <lazzari@mailup.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub.  Actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_mssql_login
+version_added: '2.7'
+short_description: adds or removes a MSSQL Server login
+description:
+    - Adds or removes MSSQL Server login
+requirements:
+  - Powershell module sqlps or Powershell module SqlServer on the remote host
+options:
+  server_instance:
+    description:
+      - Name of the sql server instance where the user has to be created
+    default: computername
+  server_instance_user:
+    description:
+      - Name of a sql user with sufficient privileges to create users
+  server_instance_password:
+    description:
+      - Password for server_instance_user
+  login_name:
+    description:
+      - Login name
+    required: yes
+  password:
+    description:
+      - Password of the new login
+  default_database:
+    description:
+      - Name of the login default database
+    default: master
+  default_language:
+    description:
+      - The login default language
+    default: us_english
+  check_expiration:
```

Comment:
Probably prefer `enforce_expiration` as a module option. Check seems to indicate whether Ansible will check something not set something.